geometry_msgs/Point32[] vertices
geometry_msgs/Point32[] normals
int32[] colors
int32[] semantic